src/securityinsight/README.md

```diff
@@ -0,0 +1,184 @@
+# Azure CLI sentinel Extension #
+This is the extension for sentinel
```

Comment:
why folder name not aligns with extension/command name?

Reply:
> why folder name not aligns with extension/command name?

@yungezz This was generated from Swagger definition.